grouper/fe/handlers.py

```diff
@@ -899,7 +912,7 @@ def post(self, group_id=None, name=None):
                 reason="Can't remove yourself. Leave group instead."
             )
 
-        group.revoke_member(self.current_user, removed_member, "Removed by owner/manager")
+        group.revoke_member(self.current_user, removed_member, "Removed by owner/np-owner/manager")
         AuditLog.log(self.session, self.current_user.id, 'remove_from_group',
                      '{} was removed from the group.'.format(removed_member.name),
                      on_group_id=group.id)
```

Comment:
This should really have an `on_user_id` for the person being removed. I know you didn't edit this code, but I just saw it and don't recall reviewing it, so.